permission-portal-frontend/client/src/components/NavBar.js

```diff
@@ -55,13 +55,13 @@ const NavBarBase = observer((props) => {
 
   return (
     <div className="navbarContainer">
-      <img src={ucsf_health} id="ucsfLogo" />
+      <img src={ucsf_health} id="ucsfLogo" alt="UCSF Health" />
```

Comment:
This will be different for each health authority, so may have to make it more generic.

Reply:
Okay, I'm having it use `store.organization.name`.